Package.swift

```diff
@@ -23,9 +24,29 @@ let package = Package(
     targets: [
         .target(
             name: "SymbolKit",
-            dependencies: []),
+            dependencies: [],
+            swiftSettings: librarySwiftSettings()
+        ),
         .testTarget(
             name: "SymbolKitTests",
             dependencies: ["SymbolKit"]),
     ]
 )
+
+func librarySwiftSettings() -> [SwiftSetting]? {
+    let manifestLocation = URL(fileURLWithPath: #filePath)
+
+    let enableLibraryEvolutionFileLocation = manifestLocation
+        .deletingLastPathComponent()
+        .appendingPathComponent(".enable-library-evolution")
+    
+    // If there is a `.enable-library-evolution` file as a sibling of this package manifest
+    // build libraries with library evolution enabled.
+    if FileManager.default.fileExists(atPath: enableLibraryEvolutionFileLocation.path) {
```

Comment:
Maybe we should add the logic to Package@swift-5.5.swift as well since most of the clients are using swift 5.5+.

Reply:
Yes! Thanks for pointing that out. Fixed